packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -236,6 +237,24 @@ export class BuildService {
     return this.prisma.build.findUnique(args);
   }
 
+  async findByRunId(runId: string): Promise<Build | null> {
+    return this.prisma.build.findFirst({
+      where: {
+        runId
+      }
+    });
+  }
+
+  async logGenerateStatusByRunId(runId: string, status: string): Promise<void> {
```

Comment:
Let's change the `status: string` type to match an enum for clarity

Reply:
added type